src/game/SpellEffects.cpp

```diff
@@ -1332,12 +1273,12 @@ void Spell::EffectTriggerSpell(SpellEffectIndex eff_idx)
         }
         // Terrordale Haunting Spirit #2, special case because chance is set to 101% in DBC while description is 55%
         case 23209:
-            if (urand(0, 100) < 55)
+            if (urand(0, 20) < 11)
                 m_caster->CastSpell(unitTarget, triggered_spell_id, true);
             return;
         // Terrordale Haunting Spirit #3, special case because chance is set to 101% in DBC while description is 35%
         case 23253:
-            if (urand(0, 100) < 35)
+            if (urand(0, 20) < 7)
```

Comment:
They however all stack up more resource usage. I'll add a comment explaining why I use the numbers that I do where I use 'obscure' numbers.